packages/client/verifiable-credentials/src/presentation/getMetadata.ts

```diff
@@ -24,11 +23,33 @@ export class ContactMetadataService {
         }
         console.debug(`Found contract ${uri} for contract ${contractAddress}`);
 
+        return this.getFromIpfs(uri);
+    }
+
+    async getContractWithVCMetadata(
```

Comment:
I find a bit of inconsistent the usage of Collection and Contract sometimes referring to the same entity. Do we have a guideling on when to use which? Why Contract here and not Collection when the return type is Collection?

Reply:
Agreed, will be addressed in next refactor